<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>481</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string>Авторизация</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>320</y>
       <width>291</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="sign_in">
        <property name="text">
         <string>Войти</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sign_up">
        <property name="text">
         <string>Зарегистироваться</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>401</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Имя </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nick"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pass"/>
      </item>
      <item>
       <widget class="QLabel" name="label_rep_pass">
        <property name="text">
         <string>Повторите пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pass_rep"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>529</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
